comparadise-utils/screenshots.ts

```diff
@@ -65,11 +65,15 @@ const trimPostfix = (path: string) => {
 };
 
 const getNewPath = (path: string) => {
-  let newPath = path.slice(path.lastIndexOf('___') + 3);
-  console.log(newPath);
-
-  if (newPath.startsWith('/')) {
-    newPath = `.${newPath}`;
+  let newPath;
+  if (path.includes('___')) {
+    newPath = path.slice(path.lastIndexOf('___') + 3);
+    if (newPath.startsWith('/')) {
+      newPath = `.${newPath}`;
+    }
+    console.log(newPath);
```

Comment:
Did you mean to remove this log?

Reply:
[Original code](https://github.com/ExpediaGroup/comparadise/pull/520/files#diff-00b199d50ad534b1d0f1281554c9296e3157db710a6032ed2f223e9430652560L69) had log statement so I kept it

Reply:
Oh sorry didn't realize 😄